src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseFilesService.cs

```diff
@@ -90,12 +90,45 @@ public class ReleaseFilesService : IReleaseFilesService
                             await _context.SaveChangesAsync();
                             await UploadFileToStorageAsync(blobContainer, releaseId, dataFile, ReleaseFileTypes.Data, dataInfo);
                             await UploadFileToStorageAsync(blobContainer, releaseId, metadataFile,ReleaseFileTypes.Metadata, metaDataInfo);
-                            await _importService.Import(dataFile.FileName.ToLower(), metadataFile.FileName.ToLower(),
-                                    releaseId, dataFile);
+                            await _importService.Import(releaseId, dataFile.FileName.ToLower(), 
+                                metadataFile.FileName.ToLower(), dataFile, false);
                             return true;
                         });
                 });
         }
+        
+        public Task<Either<ActionResult, bool>> UploadDataFilesAsZipAsync(Guid releaseId,
```

Comment:
If the Either's right is a `Unit` you don't need to return a `true`.

Reply:
done